.github/workflows/docker.yml

```diff
@@ -27,17 +31,20 @@ jobs:
       - name: set up qemu
         uses: docker/setup-qemu-action@v1
       - name: set up buildx
+        id: docker_buildx
         uses: docker/setup-buildx-action@v1
       - name: log in to ghcr
         uses: docker/login-action@v1
         with:
-          registry: ghcr.io
-          username: lus
-          password: ${{ secrets.CR_TOKEN }}
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
```

Comment:
I think this could be an issue.
GitHub's docs state that `github.actor` represents the name of the user that initiated the workflow run.
What about a merged pull request that was not merged by myself?
A constant value (maybe another custom environment variable next to `secrets.GITHUB_TOKEN`, `secrets.GITHUB_SERVICE_USER` for example) would fit the most in this case as it makes issues like that impossible to occur.

Reply:
Makes sense to me. Since secrets that aren't generated by Github can't start with `GITHUB_` I've named this secret `SERVICE_USER` (per your recommendation above) 👍